src/pages/ServiceDetails/ServiceInfo/ServiceInfoDestinationRules.tsx

```diff
@@ -14,42 +16,155 @@ class ServiceInfoDestinationRules extends React.Component<ServiceInfoDestination
+
+    return <Table.Cell className={className}>{value}</Table.Cell>;
+  };
+
+  columns() {
+    return {
+      columns: [
+        {
+          property: 'name',
+          header: {
+            label: 'Name',
+            formatters: [this.headerFormat]
+          },
+          cell: {
+            formatters: [this.cellFormat]
+          }
+        },
+        {
+          property: 'trafficPolicy',
+          header: {
+            label: 'Traffic Policy',
+            formatters: [this.headerFormat]
+          },
+          cell: {
+            formatters: [this.cellFormat]
+          }
+        },
+        {
+          property: 'subsets',
+          header: {
+            label: 'Subsets',
+            formatters: [this.headerFormat]
+          },
+          cell: {
+            formatters: [this.cellFormat]
+          }
+        },
+        {
+          property: 'host',
+          header: {
+            label: 'Host',
+            formatters: [this.headerFormat]
+          },
+          cell: {
+            formatters: [this.cellFormat]
+          }
+        },
+        {
+          property: 'createdAt',
+          header: {
+            label: 'Created at',
+            formatters: [this.headerFormat]
+          },
+          cell: {
+            formatters: [this.cellFormat]
+          }
+        },
+        {
+          property: 'resourceVersion',
+          header: {
+            label: 'Resource version',
+            formatters: [this.headerFormat]
+          },
+          cell: {
+            formatters: [this.cellFormat]
+          }
+        },
+        {
+          property: 'actions',
+          header: {
+            label: 'Actions',
+            formatters: [this.headerFormat]
+          },
+          cell: {
+            formatters: [this.cellFormat]
+          }
+        }
+      ]
+    };
+  }
+
+  yamlLink(destinationRule: DestinationRule) {
+    return (
+      <Link to={this.props.editorLink + '?destinationrule=' + destinationRule.name + '&detail=yaml'}>View YAML</Link>
+    );
+  }
+
+  rows() {
+    return (this.props.destinationRules || []).map((destinationRule, vsIdx) => ({
+      id: vsIdx,
+      name: destinationRule.name,
+      trafficPolicy: destinationRule.trafficPolicy ? (
+        <DetailObject name="" detail={destinationRule.trafficPolicy} />
+      ) : (
+        'None'
+      ),
+      subsets:
+        destinationRule.subsets && destinationRule.subsets.length > 0
+          ? this.generateSubsets(destinationRule.subsets)
+          : 'None',
+      host: destinationRule.host ? <DetailObject name="" detail={destinationRule.host} /> : undefined,
+      createdAt: <LocalTime time={destinationRule.createdAt} />,
+      resourceVersion: destinationRule.resourceVersion,
+      actions: this.yamlLink(destinationRule)
+    }));
+  }
+
+  generateKey() {
+    return (
+      'key_' +
+      Math.random()
+        .toString(36)
+        .substr(2, 9)
+    );
+  }
+
+  generateSubsets(subsets: Subset[]) {
```

Comment:
Here, as @jotak mention, is missing the displaying of a trafficPolicy for each subset.
If you see in the definition of [Subset](https://istio.io/docs/reference/config/istio.networking.v1alpha3/#Subset), it can option have a trafficPolicy.
For adding it, I would use the `<DetailObject name="trafficPolicy" detail={subset.trafficPolicy} />`.